src/CMakeLists.txt

```diff
@@ -194,6 +194,9 @@ if(BUILD_WITH_HPP_FCL_SUPPORT)
   target_compile_definitions(pinocchio_default PUBLIC PINOCCHIO_WITH_HPP_FCL)
   target_include_directories(
     pinocchio_default PUBLIC $<TARGET_PROPERTY:hpp-fcl::hpp-fcl,INTERFACE_INCLUDE_DIRECTORIES>)
+  if(COAL_DISABLE_HPP_FCL_WARNINGS)
```

Comment:
Should we add an option within the CMake for `COAL_DISABLE_HPP_FCL_WARNINGS`?

Reply:
One issue I see is user that use hpp-flc and pinocchio will never see the warning since the define will be set globally.
It's maybe better to let end-user add the `COAL_DISABLE_HPP_FCL_WARNINGS` define instead of forwarding it in the target ? I also raise the same issue with the coal PR